### PR TITLE
Add GitHub action to publish container README

### PR DESCRIPTION
**Pull request by SuperQ**
Add a GitHub action to publish the README.md to Docker Hub and Quay.io.

Fixes: https://github.com/prometheus/prometheus/issues/5348

### CODE REVIEW

**Comment by SuperQ** [comment]
My intention is to add this to the repo sync script, this is also designed to work with `prometheus-community`.

**Review by beorn7 — APPROVED** [review]
Thank you.

File: .github/workflows/container_description.yml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+          DOCKER_PASS: ${{ secrets.DOCKER_HUB_PASSWORD }}
+        with:
+          destination_container_repo: ${{ env.DOCKER_REPO_NAME }}
+          provider: dockerhub
+          short_description: ${{ env.DOCKER_REPO_NAME }}
+          readme_file: 'README.md'
+
+  PushQuayIoReadme:
+    runs-on: ubuntu-latest
+    name: Push README to Docker Hub
```
Comment (charleskorn) [inline .github/workflows/container_description.yml:35]: Should this be `Push README to Quay.io`?

Comment (SuperQ) [reply .github/workflows/container_description.yml:35]: Fixed in https://github.com/prometheus/prometheus/pull/13770

**Pull request merged**